paper/examples/flagstat/flagstat.go

```diff
@@ -0,0 +1,110 @@
+// Copyright ©2017 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// This package tabulates statistics on a bam file from the sam flag.
+// It replicates functionality in samtools flagstat.
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/biogo/hts/bam"
+	"github.com/biogo/hts/sam"
+)
+
+const (
+	pass = iota
+	fail
+
+	mask = sam.Secondary | sam.ProperPair | sam.Supplementary | sam.Unmapped
```

Comment:
Please move this const decl down to above the if statement where it is used. It makes it clearer as to how it is being used.